src/model.ts

```diff
@@ -162,7 +164,8 @@ export function convertDescribeJobtoJobDetail(
     jobId: describeJob.job_id,
     jobName: describeJob.name ?? '',
     inputFile: describeJob.input_uri,
-    outputPath: describeJob.output_uri,
+    outputPath: describeJob.output_prefix,
+    outputs: describeJob.outputs,
```

Comment:
It seems like `outputs` could be optional (`outputs?`) so that we don't need to specify it. We can treat the absence of `outputs` as not needing to display any UI

Reply:
Outputs are not optional for `DescribeJob`.